tests/modules/nf-core/goleft/indexsplit/nextflow.config

```diff
@@ -0,0 +1,8 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: GOLEFT_INDEXSPLIT {
+        ext.args = "-n 10"
```

Comment:
Is this a mandatory argument?

Reply:
yes 😈

Reply:
Well as much as it pains me to say this but can you make this an input value? 😁

Reply:
Do I have to 😞

Reply:
I'd rather not, but the guidelines say so :/

Reply:
fixed it

Reply:
Thanks!